src/createServer.js

```diff
@@ -1,3 +1,39 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('../src/convertToCase/convertToCase');
+const { validation } = require('./validation');
```

Comment:
Function 'validation' should ideally be a verb or verb phrase. Consider renaming it to 'validateInput' or similar.